src/main/java/net/education/kyivstar/config/HikariConnectionManager.java

```diff
@@ -0,0 +1,57 @@
+package net.education.kyivstar.config;
+
+import com.zaxxer.hikari.HikariDataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class HikariConnectionManager {
+    private final static Logger logger = LoggerFactory.getLogger(HikariConnectionManager.class);
+    private static HikariConnectionManager instance;
+    private final ConfigDataBase configDataBase;
+    private com.zaxxer.hikari.HikariConfig hikariConfig = new com.zaxxer.hikari.HikariConfig();
```

Comment:
Field 'hikariConfig' may be 'final'

Reply:
Why do you execute this action? What is the goal?
Optional.of(dbConfig.get(PASSWORD)).isPresent();  I check whether password is null or not,  if it is null then there is an error

Reply:
import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.SECRET_WORD;

public class PasswordEncryptor { -  I corrected it, i do not understand why  you are seeing this. Maybe I did something wrong again. I do not have this import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.SECRET_WORD;, I have this variable in this class PasswordEncryptor